src/pages/JobPosting/vacancyReview/index.tsx

```
@@ -0,0 +1,344 @@
+            </div>
+          </div>
+
+          <div className="mt-3 flex w-full max-w-[676px] items-start justify-center rounded-2xl border border-[#DEDEDE] bg-[#FFFFFF] pb-2 pl-10 pt-6 max-[391px]:relative max-[391px]:w-[358px] max-[391px]:pl-0">
+            <div className="w-[442px] pl-10 max-[391px]:max-w-[326px] max-[391px]:pl-2">
+              <h1 className="mb-6 w-[240px] text-start text-[24px] font-medium leading-[120%] max-[391px]:text-[20px]">
+                Etapas do processo seletivo
+              </h1>
+              <div className="flex flex-col items-start">
+                {formData.steps.map((o) => (
+                  <p className="mb-4 text-start text-[16px] font-light leading-[1.4] max-[391px]:text-[14px]">
+                    {o}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)